### PR TITLE
add first version of metadata JSON

### PR DESCRIPTION
**Pull request by fmalcher**

### CODE REVIEW

File: metadata.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
+{
+  "$schema": "https://angularlogos.com/assets/metadata-schema.json",
+  "logos": {
+    "angular.svg": {
+      "name": "Angular",
+      "website": "https://angular.io",
+      "license": "MIT",
```
Comment (maartentibau) [inline metadata.json]: Looking at the presskit website: https://angular.io/presskit the license is CC BY 4.0 or something.
Or maybe just CC don't really know, should we adapt is?

File: metadata.json [hunk 2]
```diff
@@ -0,0 +1,24 @@
+{
+  "$schema": "https://angularlogos.com/assets/metadata-schema.json",
```
Comment (maartentibau) [inline metadata.json:2]: Is there a way to defined in this schema that certain properties are optional?

**Review by maartentibau — CHANGES_REQUESTED** [review]

File: metadata.json (hunk 2 above)
Comment (fmalcher) [reply metadata.json:2]: Yep, all properties are optional (it works the other way around, you define the required props)

File: metadata.json (hunk 1 above)
Comment (fmalcher) [reply metadata.json]: Hmm, good question. For NOW it's not really important because this is just example data but yeah, we should not forget this!

Comment (fmalcher) [reply metadata.json]: I think we should refer to the license as said on the presskit website

**Review by maartentibau — APPROVED** [review]

**Pull request merged**